cli/src/main/java/bio/terra/pfb/JavaPfbCommand.java

```diff
@@ -21,19 +46,70 @@ static int executeCommand(String[] args) {
 
   @Override
   public void run() {
+    Library library = new Library(new PfbReader());
     System.out.println("PFB RUN");
+    if (command != null && option != null) {
+      if (command.equals(SHOW)) {
+        switch (option) {
+          case SCHEMA:
+            System.out.println("Show schema for file path: " + filePath);
```

Comment:
No need to change here, but it might be worth considering extracting all logging into a logger seam so that we could override the behavior to be something other than `System.out`.  Does this seem reasonable to you as a possible future change, or is there something that makes `System.out` a requirement?

Reply:
That is a great point -- For some reason I got in my head that I couldn't use a logger because I needed the standard output for the CLI. But, I've added a logback integration that outputs to the console. Thanks!